templates/mod/mime.conf.erb

```diff
@@ -31,6 +31,6 @@ AddLanguage sv .sv
 AddLanguage zh-CN .zh-cn
 AddLanguage zh-TW .zh-tw
 
-AddHandler type-map var
-AddType text/html .shtml
-AddOutputFilter INCLUDES .shtml
+<%- @mime_types_additional.sort.each do |key, value_hash| -%>
+        <%- value_hash.each do |value_hash_key, value_hash_value| %><%=key%> <%=value_hash_key%> <%=value_hash_value%><% end %>
```

Comment:
@igalic ya...i noticed some weird line endings too. Tried to fix in latest commit pushed 393a7fd1984e00ff9f03fe596411bd3e1e403007. please check.